Issue-2916- Provide more info about file/classes in code snippets to adding-attributes.html (#8218)
magento/devdocs: src/guides/v2.3/extension-dev-guide/extension_attributes/adding-attributes.md

</config>
```

> Here, the scalar attributes indicate the simple form of attribute representation such as an Integer or a String. The class or interface for which the extension attributes must be added is specified inside the "for" attribute of the `<extension_attributes>` tag. In this case, it is the ProductInterface. The attribute is specified with a unique code and its type.

Reviewer: ```suggestion
Here, the scalar attributes indicate the simple form of attribute representation, such as an integer or a string. Specify the class or interface of the extension attributes inside the "for" attribute of the `<extension_attributes>` tag. In this case, it is the ProductInterface. The attribute is specified with a unique code and its type.
```

> Here, the scalar attributes indicate the simple form of attribute representation such as an Integer or a String. The class or interface for which the extension attributes must be added is specified inside the "for" attribute of the `<extension_attributes>` tag. In this case, it is the ProductInterface. The attribute is specified with an unique code and it's type.

Reviewer 2: (a unique is correct as unique starts with a consonant sound)
```suggestion
Here, the scalar attributes indicate the simple form of attribute representation such as an Integer or a String. The class or interface for which the extension attributes must be added is specified inside the "for" attribute of the `<extension_attributes>` tag. In this case, it is the ProductInterface. The attribute is specified with a unique code and its type.
```

</config>
```

> The array extension attributes are just an extension of the scalar attributes where a range of values can be represented as an attribute. `[]` symbol is used to indicate the type of the attribute as array.

Reviewer 2: ```suggestion
The array extension attributes are just an extension of the scalar attributes where a range of values can be represented as an attribute. The `[]` symbol is used to indicate the type of the attribute as an array.
```

> The array extension attributes are just an extension of the scalar attributes where a range of values can be represented as an attribute. The `[]` symbol is used to indicate the type of the attribute as an array.

Reviewer: ```suggestion
The array extension attributes are just an extension of the scalar attributes where a range of values can be represented as an attribute. The `[]` symbol indicates the attribute type is an array.
```
